src/angularComponent/ngcOmniboxController.js

```diff
@@ -0,0 +1,11 @@
+import OmniboxController from 'src/coreComponent/OmniboxController.js';
```

Comment:
I disagree with this use of controller class extension. We should avoid subclassing controllers always, IMO. Angular is the only framework that even has a concept of "controllers", so it does not make sense to have a core controller file.